Exiled.Events/Patches/Events/Player/Spawning.cs

```diff
@@ -43,26 +43,29 @@ private static MethodInfo TargetMethod()
 
         private static bool Prefix(ReferenceHub hub, PlayerRoleBase prevRole, PlayerRoleBase newRole)
         {
-            if (Player.TryGet(hub, out Player player) && newRole.ServerSpawnReason != RoleChangeReason.Destroyed)
```

Comment:
Like Joker said, is this check not necessary anymore?

Reply:
I thinks it's prevent NRE and base game refuse Destroyed if i remind correctly